README.md

```diff
@@ -1,4 +1,4 @@
-# Azure IoT Edge configuration tool
+# Azure IoT Edge configuration tool [![Build Status](https://dev.azure.com/mseng/VSIoT/_apis/build/status/Azure%20IoT%20Edge/iotedgehubdev?branchName=master)](https://dev.azure.com/Azure-IoT-DDE-EdgeExperience/IoTEdgeConfig/_build?definitionId=28&branchName=main)
```

Comment:
>iotedgehubdev [](http://example.com/codeflow?start=125&length=13)

should we use the iotedgeconfig here?

Reply:
The first link is to the picture, which is only available at that link...